example/todo-app/src/main/scala/io/github/shogowada/scalajs/reactjs/example/todoapp/Main.scala

```diff
@@ -26,7 +26,7 @@ class Main {
       override def render() = {
         <.div()(
           <.h3()("TODO"),
-          <.reactElement(new TodoList(), TodoList.Props(items = state.items)),
+          new TodoList()(TodoList.Props(items = state.items)),
```

Comment:
I like the consistency of your suggestion too. In fact, I first had it like ```<(new TodoList())(TodoList.Props(items = state.items))``` just to prefix it with ```<```, but it looked weird without ```.``` so I dropped it.

The simplest I could think of is for users to create an implicit class of [```VirtualDOMElements```](https://github.com/shogowada/scalajs-reactjs/blob/96702961ca5998f5b5f69f7b9858c9cddd1e40ea/core/src/main/scala/io/github/shogowada/scalajs/reactjs/VirtualDOM.scala#L14) and add ```lazy val TodoList = new TodoList()``` so that you can ```<.TodoList(TodoList.Props(items = state.items))```. Maybe there is a way to automatically create them too if you use macros.

But then the question becomes how you can inject dependencies to the ```TodoList``` without hardcoding them, which I'd assume is a common use case in production code.

With current approach, you can just inject a instance of ```TodoList``` and use it like below, so I fallen back to this approach.

```scala
class TodoApp(todoList: TodoList) extends ReactClassSpec {
  def render() = <.div()(
    todoList(TodoList.Props(items = state.items))
  )
}

class TodoList(todoListDependency: TodoListDependency) extends ReactClassSpec {
  // ...
}
```

Of course, this can change whenever we found a way to inject dependencies to components while creating like ```<.TodoList(TodoList.Props(items = state.items))```.

Let me know if you have a concrete design in your mind too.